content/src/fieldConfig/profile.json

```diff
@@ -240,6 +240,15 @@
           "altDescription": "Do you plan to be a `home-based business|home-based-business` ?"
         }
       },
+      "elevatorOwningBusiness": {
+        "default": {
+          "header": "Elevator Owning Business",
+          "radioButtonTrueText": "Yes",
+          "radioButtonFalseText": "No",
+          "description": "Do you own a commercial or manufacturing space with an `elevator-device|elevator-device` ?",
+          "altDescription": "Do you plan to own a commercial or manufacturing space with an `elevator-device|elevator-device` ?"
```

Comment:
dust - the altDescription is using the same copy, is it needed?

Reply:
It actually isn't the same copy, the alt description talks about "plans to own" and the regular description is around "currently owns"